src/backend/distributed/sql/udfs/citus_drop_trigger/latest.sql

```diff
@@ -1,9 +1,15 @@
+CREATE OR REPLACE FUNCTION notify_constraint_dropped()
```

Comment:
should probably have a pg_catalog prefix

Reply:
addressed 09e8170